portal/src/graphql/portal/CustomDomainListScreen.module.scss

```diff
@@ -1,5 +1,5 @@
 .root {
-  padding-top: 40px;
+  margin: 0 32px;
```

Comment:
The top padding is intentional, otherwise the verify success message bar would overlap with the title. You can see it by going to `configuration/dns/custom-domains?verify=success`.

Reply:
I updated the layout to use `visibility` instead.

Reply:
I think the message bar should be full-width, instead of having spacing on two horizontal side?

Reply:
Fixed.